{"name": "test_fail", "status": "failed", "statusDetails": {"message": "AssertionError: 1 != 2", "trace": "self = <html_test_report.TestStringMethods testMethod=test_fail>\n\n    def test_fail(self):\n        \"\"\" This test should fail. \"\"\"\n>       self.assertEqual(1, 2)\nE       AssertionError: 1 != 2\n\nhtml_test_report.py:16: AssertionError"}, "description": " This test should fail. ", "start": 1645363657167, "stop": 1645363657168, "uuid": "24c9ee12-542a-405d-aea5-623cad9a2d9d", "historyId": "37a65b33b043659ce590c85936b3db86", "testCaseId": "08e49dc012b4d27b3ba3b2203469f6df", "fullName": "html_test_report.TestStringMethods#test_fail", "labels": [{"name": "suite", "value": "html_test_report"}, {"name": "subSuite", "value": "TestStringMethods"}, {"name": "host", "value": "DESKTOP-98D55J1"}, {"name": "thread", "value": "19640-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "html_test_report"}]}